index.html

```diff
@@ -36,5 +36,131 @@ <h1 class="heading"> Hey There.<br>I’m Ismail</h1>
             <li><a href="#" alt="meta"><img src='Assets/images/meta.svg' alt="meta"/></a></li>
         </ul>
     </section>
+    <section class="works-section">
+        <h2 class="work-heading">My Recent Works</h2>
+        <section class="post">
+            <img src="Assets//images/post.svg" alt="post image"/>
+            <article class="article">
+                <h3 class="post-heading">Multi-Post Stories</h3>
+                <p class="post-info">A daily selection of privately personalized reads;
+                no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            </article>
+            <ul class="lang-btn">
+                <li><a href="#" alt="css">css</a></li>
+                <li><a href="#" alt="html">html</a></li>
+                <li><a href="#" alt="bootstrap">bootstrap</a></li>
+                <li><a href="#" alt="ruby">Ruby</a></li>
```

Comment:
- [ ] There are not links, it's just a normal list of the technologies used in the project so it's better if you remove the `a` tag wrapping each text (this applies to all the cards) 💯